docs/source/guide/distributed-partition.rst

```diff
@@ -69,6 +69,33 @@ second dictionary contains the mapping for each edge type.
     orig_node_emb = th.zeros(node_emb.shape, dtype=node_emb.dtype)
     orig_node_emb[node_map] = node_emb
 
+As a counterpart of ``return_mapping=True`` in :func:`~dgl.distributed.partition_graph`,
+below 2 arguments are provided for distributed partition pipeline in ``dispatch_data.py``.
+
+* ``--save-orig-nids`` save original node IDs into files.
+* ``--save-orig-eids`` save original edge IDs into files.
+
+Original node/edge IDs are split into the number of partitions. So these
+files lie under the same path of graph partitions and node/edge feature data.
+Below code shows how to obtain the same node/edge mappings of
+:func:`~dgl.distributed.partition_graph` shown above.
+
+.. code:: python
+
+    node_map = []
+    edge_map = []
+    for i in range(num_parts):
+        part_orig_nids = dgl.data.load_tensors(f'/tmp/test/part{i}/orig_nids.dgl')['_N']
+        node_map.append(part_orig_nids)
+        part_orig_eids = dgl.data.load_tensors(f'/tmp/test/part{i}/orig_eids.dgl')['_E']
+        edge_map.append(part_orig_eids)
+    node_map = torch.cat(node_map)
+    edge_map = torch.cat(edge_map)
```

Comment:
Users may not want to concatenate all the node IDs. I will make the example code extremely simple:
```python
# Load the original IDs for the nodes in partition 0.
orig_nids_0 = dgl.data.load_tensors('/path/to/data/part0/orig_nids.dgl')
# Get the original node IDs for node type 'user'
user_orig_nids_0 = orig_nids_0['user']

<same for orig eids>
```